database/schema.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE label (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(150),
+    color VARCHAR(150)
+);
+
+CREATE TABLE book (
+    id INTEGER PRIMARY KEY,
+    publisher VARCHAR(100),
+    publish_date DATE,
+    cover_state VARCHAR(100),
+    item_id INTEGER,
+    FOREIGN KEY (item_id) REFERENCES item (id)
+);
```

Comment:
This SQL statement so far is making reference to an Item table that does not exist yet. I think it will be better to have a complete set of statements to create all the tables and thier relationships than just having one thatsis assuming that a table called item with an id column will be eventually created.